site/docs/status.md

```diff
@@ -0,0 +1,362 @@
+---
+title: "Implementation Status"
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Implementations Status
+
+Apache iceberg now has implementations of the iceberg spec in multiple languages. This page provides a summary of the
+current status of these implementations.
+
+## Versions
+
+This section describes the versions of each implementation that are being tracked in this page.
+
+| Language  | Version |
+|-----------|---------|
+| Java      | 1.7.1   |
+| PyIceberg | 0.7.0   |
+| Rust      | 0.3.0   |
+| Go        | 0.1.0   |
+
+## Data Types
+
+| Data Type      | Java | PyIceberg | Rust | Go |
+|----------------|------|-----------|------|----|
+| boolean        | Y    | Y         | Y    | Y  |
+| int            | Y    | Y         | Y    | Y  |
+| long           | Y    | Y         | Y    | Y  |
+| float          | Y    | Y         | Y    | Y  |
+| double         | Y    | Y         | Y    | Y  |
+| decimal        | Y    | Y         | Y    | Y  |
+| date           | Y    | Y         | Y    | Y  |
+| time           | Y    | Y         | Y    | Y  |
+| timestamp      | Y    | Y         | Y    | Y  |
+| timestamptz    | Y    | Y         | Y    | Y  |
+| timestamp_ns   | Y    | Y         | Y    | Y  |
+| timestamptz_ns | Y    | Y         | Y    | Y  |
+| string         | Y    | Y         | Y    | Y  |
+| uuid           | Y    | Y         | Y    | Y  |
+| fixed          | Y    | Y         | Y    | Y  |
+| binary         | Y    | Y         | Y    | Y  |
+
+## Data File Formats
+
+| Format  | Java | PyIceberg | Rust | Go |
+|---------|------|-----------|------|----|
+| Parquet | Y    | Y         | Y    | Y  |
```

Comment:
we should include Avro aswell? 
https://iceberg.apache.org/docs/nightly/configuration/#write-properties

Reply:
I think there was some confusion, I think we should split out the data and metadata part. In this table, I would suggest adding Avro as a data file format. Then we need another Avro, that's similar to the Puffin discussion below:

1. Basic support for Avro format, e.g. read/write capability, and this is what the file format section means.